azure-pipelines.yml

```diff
@@ -40,8 +40,8 @@ jobs:
         env:
           CI: 1  # Used to skip certain tests on Azure
           SHOTGUN_HOST: $(sg.ci.host)
-          SHOTGUN_SCRIPT_NAME: $(shotgun_script_name)
-          SHOTGUN_SCRIPT_KEY: $(shotgun_script_key)
+          SHOTGUN_SCRIPT_NAME: $(sg.ci.script.name)
+          SHOTGUN_SCRIPT_KEY: $(sg.ci.script.key)
```

Comment:
Are these lines supposed to have changed? Is this to do with the key rotation work?

Reply:
They needed to be changed to fix the tests. [JF has renamed the Azure secrets](https://github.com/shotgunsoftware/tk-ci-tools/pull/20) to make the names consistent and we hadn't updated this repo since that change was made.